dclab/definitions/feat_const.py

```diff
@@ -126,6 +130,11 @@
     ["mask", "Binary mask labeling the event in the image"],
     # See FLUOR_TRACES for valid keys
     ["trace", "Dictionary of fluorescence traces"],
+    # QPI features computed from holographic data
+    ["qpi_oah", "Off axis hologram"],
+    ["qpi_oah_bg", "Off axis hologram background image"],
```

Comment:
have to wait to see if we need this or can compute pha_bg and amp_bg

Reply:
Yes we can compute it with `rollmed` or `sparsemed`.

`sparsemed` is much faster with no drop in accuracy. Note that it doesn't work for very small datasets as it is based on measurement time.

Reply:
@paulmueller I wasn't sure how to phrase this, or if it is even the approach you would take. See line: https://github.com/DC-analysis/dclab/pull/227/files#diff-bf062dd8b1aff922e4223a0a780a7e16928551f4355f269d18426c1e706b2ce5R135

Reply:
I think "experimental" might be ambiguous. Better use "direct", "camera", or "empty frame" or somesuch?
Are you planning to perform more elaborate background correction schemes (e.g. tilt correction) or are those off the table?